spec.md

```diff
@@ -58,6 +58,14 @@ that does not support that feature will then silently ignore that part of the
 message. The sender needs to be prepared for the situation where a receiver
 ignores that feature.
 
+For clarity, when a feature is marked as "OPTIONAL" this means that it is
```

Comment:
I think the edits in this doc are invalid

Reply:
@duglin  - I believe I have applied the changes correctly ;) - would you mind reviewing?